packages/lesswrong/components/languageModels/LanguageModelChat.tsx

```diff
@@ -193,77 +357,77 @@ const LLMInputTextbox = ({onSubmit, classes}: {
     setCurrentMessage('');
   }, [onSubmit]);
 
-  // We need to pipe through the `conversationId` and do all of this eventListener setup/teardown like this because
-  // otherwise messages get submitted to whatever conversation was "current" when the editor was initially loaded
-  // Running this useEffect whenever either the conversationId or onSubmit changes ensures we remove and re-attach a fresh event listener with the correct "targets"
-  useEffect(() => {
-    const currentEditorRefValue = ckEditorRef.current;
-
-    const options = { capture: true };
-    const handleKeyDown = (event: KeyboardEvent) => {
-      if ((event.ctrlKey || event.metaKey) && event.keyCode === 13) {
-        event.stopPropagation();
-        event.preventDefault();
-        submitEditorContentAndClear();
-      }
-    };
-  
-    const internalEditorRefInstance = (currentEditorRefValue as AnyBecauseHard).domContainer?.current;
-    if (internalEditorRefInstance) {
-      internalEditorRefInstance.addEventListener('keydown', handleKeyDown, options);
+  useGlobalKeydown((event: KeyboardEvent) => {
```

Comment:
I assume this change was motivated by something or other, but not clear it's desirable, we should discuss